neurokit2/signal/signal_detrend.py

```diff
@@ -134,7 +134,7 @@ def _signal_detrend_tarvainen2002(signal, regularization=500):
     """
     N = len(signal)
     identity = np.eye(N)
-    B = np.dot(np.ones((N - 2, 1)), np.array([[1, -2, 1]]))
+    B = np.dot(np.ones((N, 1)), np.array([[1, -2, 1]]))
     D_2 = scipy.sparse.dia_matrix((B.T, [0, 1, 2]), shape=(N - 2, N))  # pylint: disable=E1101
```

Comment:
Interesting, so they suggest changing that line to:

```suggestion
    D_2 = scipy.sparse.dia_matrix((B.T, [0, 1, 2]), shape=(N - 3, N - 1))
```

Reply:
Please do give it a test. Does this change a lot in "normal" scenarios (i.e., would user that used the previous version have significantly different results after the change)? Just to know wether to label that change as a fix or a breaking change